popup.html

```diff
@@ -35,28 +35,7 @@
 				min-height: 60px;
 				resize: vertical;
 			}
-			.input-group textarea#numbers-and-names {
```

Comment:
i've noticed that some of these selectors are referencing code that's commented out further down in the file.  i think before wholesale deleting this, we should check out what that commented out code is doing, and if it's not functionality that we foresee as being useful, then we should delete that too, lest we run the risk of trying to reintroduce it later only to be confused as to why the UI is messed up.  there are also some calls to `document.getElementById` in popup.js that are referencing ids from this commented out code.